research/conditional/train/configs/test/test_baseline.yaml

```diff
@@ -30,4 +30,7 @@ params:
   lr_decay: 0.8
   lr_warmup_steps: 0
   lr_decay_interval: 10000
+  n_att_heads: 4
+  learning_rate: 0.0001
```

Comment:
default jest inny, więc to się łączy z taskiej ustalania test_baseline'u.
Zmiana poza scope'm

Reply:
Ale to jest tylko na unittesty, tak? To wtedy nie ma znaczenia, jak mniej więcej się zgadza. Tylko najwyżej dla pewności bym trzymał dmodel/dff jako śmiesznie mały (16/64), żeby nikt tego przypadkiem nie użył serio.

Reply:
to unit test